src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -732,7 +732,7 @@ private void SelectVersionPackageReferenceProject(NuGetVersion latestVersion)
             }
             else
             {
-                var installedVersion = _searchResultPackage?.AllowedVersions?.OriginalString ?? _searchResultPackage?.InstalledVersion.ToString();
+                var installedVersion = _searchResultPackage?.AllowedVersions?.OriginalString ?? _searchResultPackage?.InstalledVersion.ToNormalizedString();
```

Comment:
In the checklist, you listed "N/A" for testing. Why do you believe testing is not applicable?

I know many parts of NuGet are not written in ways that are testable, especially the VS parts. So I'd believe a test exception (not n/a) saying it's not testable. But, we should always look for opportunities to write automated tests that can prevent regressions.